mods/creative/inventory.lua

```diff
@@ -59,22 +59,34 @@ function creative.init_creative_inventory(player)
 	return player_inventory[player_name]
 end
 
+local function match(s, filter)
+	if filter == "" then return 0 end
```

Comment:
I think we should not accept 1 line conditionals like this, even if that is not stated in codestyle guidelines, it is strongly discouraged in linux kernel guidelines.

Reply:
_"Brevity is the Soul of Wit"_